### PR TITLE
replace pkg/errors

### PR DESCRIPTION
**Pull request by zouyee**
Signed-off-by: Zou Nengren <zouyee1989@gmail.com>

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Review by Zyqsempai — APPROVED** [review]
LGTM

**Review by estesp — APPROVED** [review]
LGTM

File: plugin/plugin.go [hunk 1]
```diff
@@ -50,7 +52,7 @@ func init() {
 			ic.Meta.Exports["root"] = root
 			snapshotter, err := zfs.NewSnapshotter(root)
 			if err != nil {
-				return nil, errors.Wrap(plugin.ErrSkipPlugin, err.Error())
+				return nil, fmt.Errorf("%s: %w", plugin.ErrSkipPlugin.Error(), err)
```
Comment (fuweid) [inline plugin/plugin.go:55]: Is it possible to use `errors.Is(err, plugin.ErrSkipPlugin)` after this change?

Comment (fuweid) [reply plugin/plugin.go:55]: @zouyee

Comment (fuweid) [reply plugin/plugin.go:55]: It should be `fmt.Errorf("%w: %s", plugin.ErrSkipPlugin, err.Error())` right?

**Pull request merged**